components/brave_wallet_ui/components/desktop/asset-watchlist-item/index.tsx

```diff
@@ -93,7 +92,7 @@ const AssetWatchlistItem = React.forwardRef<HTMLDivElement, Props>(
     return (
       <StyledWrapper ref={forwardedRef}>
         <NameAndIcon onClick={onClickAsset}>
-          {token.isErc721 && !isDataURL(token.logo) ? (
+          {token.isNft ? (
```

Comment:
I thought `isNft` was reserved for Solana (please correct me if not the case).

If the above is true, a more comprehensive condition would be:

```ts
token.isNft || token.isErc721 || token.isErc1155
```

Reply:
`isNFT` Works for everything: https://github.com/brave/brave-browser/issues/26395

Reply:
[Auto detect token standard when adding NFT assets](https://github.com/brave/brave-core/pull/17207#top)